VizWiz.Dockerfile

```
@@ -0,0 +1,67 @@
+FROM nvidia/cuda:11.8.0-devel-ubuntu22.04
+LABEL name="unified-io-inference"
+
+WORKDIR /root/.conda
+WORKDIR /root
+RUN apt-get update && apt-get -y install wget nano
+RUN wget \
+    https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh 
+ENV PATH=/opt/conda/bin:${PATH}
+RUN bash -c "conda update -n base -c defaults conda"
+
+RUN wget -nv https://ai2-prior-uio.s3.us-west-2.amazonaws.com/public/model-weights-bin/xl_1000k.bin \
+ -O xl.bin
+RUN wget -nv https://ai2-prior-uio.s3.us-west-2.amazonaws.com/public/model-weights-bin/large_1000k.bin \
+ -O large.bin
+RUN wget -nv https://ai2-prior-uio.s3.us-west-2.amazonaws.com/public/model-weights-bin/base_1000k.bin \
+ -O base.bin
+RUN wget -nv https://ai2-prior-uio.s3.us-west-2.amazonaws.com/public/model-weights-bin/small_1000k.bin \
+ -O small.bin
+RUN wget -nv https://farm2.staticflickr.com/1362/1261465554_95741e918b_z.jpg -O dbg_img.png
+
+COPY uioi.yml .
+RUN bash -c "conda env create -f uioi.yml"
+COPY requirements.txt .
+RUN bash -c ". activate uioi && pip install --upgrade pip \
+ && pip install --upgrade "jax[cuda]" \
+ -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html \
+ && python3 -m pip install -r requirements.txt"
+
+RUN bash -c ". activate uioi && pip install matplotlib notebook numpy==1.23.5 nvidia-cudnn-cu11==8.6.0.163 setuptools wheel spacy webdataset && python3 -m spacy download en_core_web_sm"
+
+ENV PYTHONPATH=/root/uio
+RUN apt-get -y install openjdk-8-jdk 
+RUN apt-get update && apt-get -y upgrade && apt-get -y install unzip 
+RUN apt-get update && apt-get -y upgrade && apt-get -y install git
```

If we're interested in minimizing the number of layers, could these three `apt-get install` lines be merged?

```suggestion
RUN apt-get update && apt-get -y upgrade && apt-get -y install git openjdk-8-jdk unzip
```

---

Yes, it would be better if they were merged.  In my limited testing, I got a build error when some of these were combined.  Not sure what the issue was, exactly.